nengo_bones/tests/test_generate_bones.py

```diff
@@ -262,3 +262,41 @@ def test_custom_template(tmpdir):
         data = f.read()
 
     assert "; then\n\n    this is a custom message\n\nelif" in data
+
+
+def test_empty_codecov_yml(tmpdir):
+    # minimal config, testing defaults
+    write_file(tmpdir, ".nengobones.yml", """
+            pkg_name: dummy
+            repo_name: dummy/dummy_repo
+            travis_config:
+              jobs:
+                - thing: val
+            """)
+
+    result = CliRunner().invoke(
+        generate_bones.main,
+        ["--conf-file", str(tmpdir.join(".nengobones.yml")),
+         "--output-dir", str(tmpdir),
+         "codecov-yml"])
+    assert_exit(result, 1)
+
+    assert str(result.exception) == "'codecov_config'"
+
+
+def test_empty_travis_yml(tmpdir):
+    # minimal config, testing defaults
+    write_file(tmpdir, ".nengobones.yml", """
+            pkg_name: dummy
+            repo_name: dummy/dummy_repo
+            codecov_yml: {}
```

Comment:
This should be `codecov_config` (although it doesn't really matter in this case).

Reply:
This actually makes me think though, it'd probably make more sense to make these headings have the same name as the `generate-scripts` command (i.e. `ci_scripts`, `travis_yml`, and `codecov_yml`). Does that sound good to you?

Reply:
Done (change will be applied later to the other repos).